package-lock.json

```diff
@@ -7935,6 +7916,26 @@
 				}
 			}
 		},
+		"requireg": {
```

Comment:
Could you please update `package-lock.json` to not include requireg? A simple `npm install` should do the trick.